tests/test_target_scheduling.py

```
@@ -0,0 +1,79 @@
+introduced in MTV 2.10.0. Tests are skipped if MTV version < 2.10.0.
+"""
+
+import pytest
+from pytest_testconfig import config as py_config
+
+from utilities.mtv_migration import create_storagemap_and_networkmap, migrate_vms
+from utilities.utils import has_mtv_minimum_version
+
+
+@pytest.mark.parametrize(
+    "plan,multus_network_name",
```

Ruff (0.14.11)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)